README.md

```diff
@@ -11,7 +11,7 @@ Bash script used to compile PHP on MacOS and Linux platforms. Make sure you have
 #### Mac OSX (native compile)
 - Most dependencies can be installed using Homebrew
 - You will additionally need `glibtool` (GNU libtool, xcode libtool won't work)
-- You also MUST specify target as `mac` or `mac64` if building for Mac, on Mac.
+- You also MUST specify target as `mac-x86-64` or `mac-arm64` if building for Mac, on Mac.
```

Comment:
This line can be removed entirely. Auto-detection now works for macOS.

Reply:
Oh brilliant!